deploy.sh

```diff
@@ -82,8 +82,7 @@ if [ "$PUBLISH" = true ]; then
   sbt +publishSigned
   # Upload to Central Portal and auto-release
   sbt sonaUpload
-  # TODO: should be do auto-release?
-  # sbt sonaRelease
+  sbt sonaRelease
```

Comment:
Currently we need release team to manually publish artifacts from the maven. With the change, the publish step becomes automatic. No need for human to do it manually, which aligns with many other Snowflake client releases